util/regexp-assemble/regexp-942362.data

```diff
@@ -0,0 +1,134 @@
+alter\s+char\s?[(]?
+alter\s+group_concat\s?[(]?
+alter\s+load_file\s?[(]?
+create\s+char\s?[(]?
+create\s+group_concat\s?[(]?
+create\s+load_file\s?[(]?
+delete\s+char\s?[(]?
+delete\s+group_concat\s?[(]?
+delete\s+load_file\s?[(]?
+desc\s+char\s?[(]?
+desc\s+group_concat\s?[(]?
+desc\s+load_file\s?[(]?
+insert\s+char\s?[(]?
+insert\s+group_concat\s?[(]?
+insert\s+load_file\s?[(]?
+load\s+char\s?[(]?
+load\s+group_concat\s?[(]?
+load\s+load_file\s?[(]?
+rename\s+char\s?[(]?
+rename\s+group_concat\s?[(]?
+rename\s+load_file\s?[(]?
+select\s+char\s?[(]?
+select\s+group_concat\s?[(]?
+select\s+load_file\s?[(]?
+truncate\s+char\s?[(]?
+truncate\s+group_concat\s?[(]?
+truncate\s+load_file\s?[(]?
+update\s+char\s?[(]?
+update\s+group_concat\s?[(]?
+update\s+load_file\s?[(]?
+end\s*?\);
+[\s(]load_file\s*?\(
+[\"'`]\s+regexp\W
+[\d\W]\s+as\b\s*[\"'`\w]+\s*\bfrom
+^[\W\d]+\s*?create\s+\w+
+^[\W\d]+\s*?delete\b
+^[\W\d]+\s*?desc\b
+^[\W\d]+\s*?insert\b
+^[\W\d]+\s*?load\b
+^[\W\d]+\s*?rename\b
+^[\W\d]+\s*?select\b
+^[\W\d]+\s*?truncate\b
+^[\W\d]+\s*?update\b
+^[\W\d]+\s*?alter\s*aggregate\b
```

Comment:
Why do we double these regexes from here downwards, I mean these `alter...`, into the rule 942362? 
And also line 32 (`load_file`), line 33 (`regexp`) and line 35 (`create`) while they don't change.
Don't we just want to double the regexes into 942362 that get stricter?
Just asking, I'm not sure...

Reply:
No strong preferences here, I just kept the old 942360 as is in order to keep the review simple(r).

Reply:
I see. Personally, I wouldn't duplicate them. But also no strong preferences here. Other opinions?

Reply:
Original suggestion to duplicate the old rule came from @dune73:
https://github.com/coreruleset/coreruleset/pull/1817#issuecomment-740594936

Reply:
Initially I thought this would play out better with strict siblings, but in fact it does not really bring any advantages here. On top the lower PL rule is executed anyway. So I think it can be skipped.

But maybe we can have @lifeforms opinion as well. I would like to make sure we get this right.